### PR TITLE
Some pylint cleanups for gramps/gen/plug/*.py

### PR DESCRIPTION
**Pull request by cdhorn**

### CODE REVIEW

File: gramps/gen/plug/_manager.py [hunk 1]
```diff
@@ -193,10 +191,7 @@ def reg_plugins(
                 if count > max_count:
                     print("Cannot resolve the following plugin dependencies:")
                     for plugin in plugins_to_load:
-                        print(
-                            "   Plugin '%s' requires: %s"
-                            % (plugin.id, plugin.depends_on)
-                        )
+                        print("   Plugin '{plugin.id}' requires: {plugin.depends_on}")
```
Comment (Nick-Hall) [inline gramps/gen/plug/_manager.py:194]: Should this be an f-string?

File: gramps/gen/plug/_manager.py [hunk 2]
```diff
@@ -209,10 +204,10 @@ def reg_plugins(
                     try:
                         results = mod.load_on_reg(dbstate, uistate, plugin)
                     except:
-                        import traceback
+                        import traceback  # pylint: disable=import-outside-toplevel
 
                         traceback.print_exc()
-                        print("Plugin '%s' did not run; continuing..." % plugin.name)
+                        print("Plugin '{plugin.name}' did not run; continuing...")
```
Comment (Nick-Hall) [inline gramps/gen/plug/_manager.py:210]: Another f-string?

File: gramps/gen/plug/_options.py [hunk 3]
```diff
@@ -228,33 +228,27 @@ def save(self):
...
                         file.write(
-                            "  <option name=%s value=%s/>\n"
-                            % (quoteattr(option_name), quoteattr(str(option_data)))
+                            f"  <option name={quoteattr(option_name)} "
+                            f"value={quoteattr(str(option_data))}/>\n"
                         )
                 for docgen_name in self.docgen_names:
                     if module_docgen_opts[docgen_name]:
-                        for idx, data in enumerate(module_docgen_opts[docgen_name]):
+                        for _, data in enumerate(module_docgen_opts[docgen_name]):
```
Comment (Nick-Hall) [inline gramps/gen/plug/_options.py:247]: Can we remove the `enumerate` here if the index is not needed?

**Review by Nick-Hall — CHANGES_REQUESTED** [review]
Only a few things to check on this one.

Our policy is not to include pylint comment statements in the code.

**Comment by cdhorn** [comment]
@Nick-Hall this should be ready now.

**Review by Nick-Hall — APPROVED** [review]

**Pull request merged**